drivers/scale_service.go

```diff
@@ -40,6 +40,10 @@ func (s *ScaleServiceDriver) ValidatePayload(conf interface{}, apiClient client.
 		return http.StatusInternalServerError, errors.Wrap(err, "Error in getService")
 	}
 
+	if service.Kind != "service" {
+		return http.StatusBadRequest, fmt.Errorf("Cannot create webhooks for External Service or Service Alias")
```

Comment:
Because there are more than these two types (External Svc and Svc Alias) and there could be even more in the future, the message shouldn't mention specific types.
You could change to something like
```
Can only create webhooks for Services. The supplied service is of type %v
```